cluster-controller/src/main/java/io/strimzi/controller/cluster/resources/KafkaConnectCluster.java

```diff
@@ -123,7 +142,7 @@ public static KafkaConnectCluster fromConfigMap(ConfigMap cm) {
     public static KafkaConnectCluster fromDeployment(K8SUtils k8s, String namespace, String cluster) {
 
         Deployment dep = k8s.getDeployment(namespace, cluster + KafkaConnectCluster.NAME_SUFFIX);
-
+log.info("Namespace: {}, Cluster: {}", namespace, cluster);
```

Comment:
Wrong indentation. Was this line just for tracing during your tests or you want to leave it ? We don't have this kind of log in the other fromStatefulSet methods.

Reply:
Sorry, this was just a debug statement when debuging the issues from #183